### PR TITLE
[Doc Improvement][Register bot in developer portal for SSO][2607461]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80d6c37c686d76ec06718e313eb6a6a24f8674b6/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f5afbb8-dd8c-4e67-bbc9-e2d07da037b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/99b34d952bfc12ceedace63aa89a32637196165a/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eefb5728-eefc-4ef6-b3b7-948e1bf95760) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d779473b93d358114d41b1f2af04ed7e25228e8/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/82f6ac59-cec9-4c6e-b301-19217b13107c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/12676629d62ba04aaac2de7ad43c04fd36dc2f92/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a983cc4-7a55-4ba0-a0ef-16b2fbaf6664) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33d6046f653fd42a973187b72cb035d0b817c429/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2bf144e-408e-4e12-a88e-f5c104bca63e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/38c2b5a7ff2b9235100cf8d725566c6428fcbfec/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84007a71-c61e-4a4f-a651-ca43918b48ad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1e0eed9ed7be29f9f5946616f16289684709e696/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 95 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f0f30017-2653-47fe-8a17-81dda1659389) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ In the **Configure** section, you can see the following components to manage and
   * Meeting extension
   * Activity feed notification
 * **Permissions**: This section allows you to give Device permissions, Team permissions, Chat or Meeting permissions, and User Permissions for your app.
-* **Single sign-on**: You can configure your app to authenticate users with single sign-on (SSO).
+* **Single sign-on**: Developer Portal supports the Single sign-on (SSO) configured for the Teams app in Azure AD. Add the **Application ID URI** that you configured for your app in the Azure AD.
```
Comment (osheenkhare) [inline msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md:54]: this is just the manifest field for SSO. We should be mentioning that Bot Registered on Azure AD supports Single Sign on. If a bot is registered on Bot Framework Portal (or within the developer Portal under Bot Management) then those bots don't support SSO and have to be Registered on Azure for them to Support SSO.

For a bot registered on Azure, add the Application ID URI, Refer to [this link](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/authentication/auth-aad-sso-bots) on how to get Application URI

Comment (osheenkhare) [reply msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md:54]: Developer Portal supports the Single sign-on (SSO) : Please remove this line as developer portal does not support SSO, it only ask for the user to input the Application URI for Bots provisioned on Azure

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c41d31aef343c6f8cc23ad8c10dfbc317f71f51/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50b36ea9-6be4-4eff-82b5-3c7e77a4a56c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by osheenkhare — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/72a60eb743ecfccbbd489ce0c740b07af8efb26d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0d35353-8fef-425c-9eca-38caca30bde9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**